content/en/docs/guides/dualstack.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Dualstack Backends"
+description: >-
+     Explains the dualstack backend feature in gRPC.
+---
+
+### Overview
+
+In cases where an endpoint (i.e., a gRPC server) has both an IPv4 and
+IPv6 address, it is useful for clients to know both of those addresses,
+so that they can reach the endpoint on either of them.  Because only
+one of the two protocols may actually be routable between a given
+client and endpoint, clients need to try both addresses to see which
+one will work.  To minimize connection attempt latency, it is also
+desirable for clients to attempt to connect to multiple addresses in
+parallel, which is done using the "Happy Eyeballs" algorithm described in
+[RFC-8305](https://www.rfc-editor.org/rfc/rfc8305).
+
+In gRPC, it is particularly important for the [Load Balancing
+Policy](https://grpc.io/docs/guides/custom-load-balancing) to understand
+which addresses are associated with the same endpoint as opposed to
+being different endpoints, so that the client does not send twice as
+much load to a endpoint with two addresses.  To that end, the [Name
+Resolver](https://grpc.io/docs/guides/custom-name-resolution) can return
+multiple addresses per endpoint (where an endpoint is a single endpoint),
+and that information is communicated to the Load Balancing Policy.
+
+Note that the only name resolver implementation that gRPC provides is a
+DNS resolver, which does not support dualstack backends.  This is
```

Comment:
It won't be clear to users what "does not support" means here. You can use dualstack backends with pick_first. Maybe the earlier paragraph about load balancing policies needs to call out that nothing special needs to happen for the default policy, pick_first. But if using a different policy it probably needs NR support? And then you can use round_robin as an example when explaining the twice-the-load problem.